manifests/pe_metric.pp

```diff
@@ -46,6 +46,13 @@
     command => "find '${output_dir}' -type f -mtime ${retention_days} -delete",
   }
 
+  cron { "${metrics_type}_metrics_tar" :
+    ensure  => $metric_ensure,
+    user    => 'root',
+    hour    => '1',
+    command => "export DIR='${output_dir}' SVC=${metrics_type} ; find \"\$DIR/\$SVC\" -type f \! -name \"*.bz2\" > \"\$DIR/\$SVC.tmp\" ; xargs -a \"\$DIR/\$SVC.tmp\" tar -jcf \"\$DIR/\$SVC/\$SVC-$(date +%Y%m%d).tar.bz2\" 2>/dev/null ; xargs -a \"\$DIR/\$SVC.tmp\" rm ; rm \"\$DIR/\$SVC.tmp\"",
```

Comment:
wow that's more complicated than I was expecting.  I think people would like it better if we combined this with the _metrics_tidy cron though.